docs/examples/gluster-s3-storage-template/README.md

```diff
@@ -2,111 +2,132 @@
 
 ## Prerequisites
 
-  OpenShift setup is up with master and nodes ready.
+OpenShift setup is up with master and nodes ready.
 
-  cns-deploy tool is ran and heketi service is ready
+cns-deploy tool is ran and heketi service is ready.
 
+## First create a storageclass
```

Comment:
Insert a line above this one:
```
## Deployment
```
Then change this line to:
```
### 1. Provide the backend store
```

Reply:
Done